src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -365,6 +365,9 @@ private void filmSuchen2(String urlFilm_, String thema, String titel, long dauer
                     }
                 }
                 for (String s : liste) {
+                    if(s.startsWith("//")) {
+                        s = "http:" + s;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "http:" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)